scanner/src/main/kotlin/storages/PostgresStorage.kt

```diff
@@ -36,11 +36,9 @@ import com.vdurmont.semver4j.Semver
 
 import java.io.IOException
 import java.sql.Connection
-import java.util.SortedSet
 
-class PostgresStorage(private val connection: Connection) : ScanResultsStorage() {
-    private val schema = "public"
-    private val table = "scan_results"
+class PostgresStorage(val connection: Connection, val schema: String) : ScanResultsStorage() {
+    val table = "scan_results" // TODO: make configurable
```

Comment:
This should stay `private`. (And the `TODO` maybe be moved to the previous commit.)

Reply:
Done.